example-android/app/build.gradle

```diff
@@ -65,12 +66,18 @@ android {
             }
         }
     }
+
+    compileOptions {
```

Comment:
Please avoid setting java version. It will require bumping major version which is not desired at this point.